cmd/format.go

```diff
@@ -11,20 +12,20 @@ import (
 
 var formatCmd = &cobra.Command{
```

Comment:
Kan ønske meg at man legger til et alias for `format`, nemlig `fmt` iom. det er relativt utbredt i lignende tooling.